core-build/gulp-core-build/src/tasks/JestTask.ts

```diff
@@ -136,7 +141,16 @@ export class JestTask extends GulpTask<IJestConfig> {
       moduleDirectories: this.taskConfig.moduleDirectories ?
         this.taskConfig.moduleDirectories :
         ['node_modules', this.buildConfig.libFolder],
-      reporters: [path.join(__dirname, 'JestReporter.js')],
+      reporters: [
```

Comment:
I was torn between making two separate reporters and using the task config to swap between the two, but the existing reporter was so barebones that it made sense to build it in and optionally enable the actual output within the reporter... but that's just my line of thinking